src/core/server/logging/README.md

```diff
@@ -321,3 +323,20 @@ Define a custom logger for a specific context.
 
 #### logging.filter
 TBD
+
+### Log record format changes
+
+| Parameter       | Platform log record in **pattern** format  | Legacy Platform log record **text** format |
+| --------------- | ------------------------------------------ | ------------------------------------------ |
+| date            | ISO8601 `2012-01-31T23:33:22.011Z`         | Absolute `23:33:22.011`                    |
+| context         | `parent.child`                             | `['parent', 'child']`                      |
+| level           | `DEBUG`                                    | `['debug']`                                |
+| meta            | stringified JSON object `{"to": "v8"}`     | N/A                                        |
+| pid             | can be configured as `%pid`                | N/A                                        |
+
+| Parameter       | Platform log record in **json** format     | Legacy Platform log record **json** format   |
+| --------------- | ------------------------------------------ | -------------------------------------------- |
+| date            | ISO8601_TZ `2012-01-31T23:33:22.011-05:00` | ISO8601 `2012-01-31T23:33:22.011Z`           |
+| context         | `context: parent.child`                    | `tags: ['parent', 'child']`                  |
+| level           | `level: DEBUG`                             | `tags: ['debug']`                            |
+| meta            | separate property `"meta": {"to": "v8"}`   | merged in log record  `{... "to": "v8"}`     |
```

Comment:
No pid? We did have pid in legacy json format

Reply:
the table documents **changes**, so pid is omitted. Do you think it's worth adding all the params?

Reply:
Yeah this might be slightly more useful if it was just a complete table, including unchanged fields.